src/components/Missions.js

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Table } from 'react-bootstrap';
+import Mission from './Mission';
+import { getMisstions } from '../redux/missions';
```

Comment:
[OPTIONAL] I think you could be more careful when spelling different words. Although the code might work because it's named the same everywhere. If later you want to come back or if another programmer has to work with your code it will have a hard time.